src/sequelize/relations.ts

```diff
@@ -0,0 +1,128 @@
+export function createRelations(db) {
+  /**
+   * ACCESSORIE
+   */
+  db.accessorie.belongsToMany(db.offer,  {
```

Comment:
Sequelize adds 's' to the table names, i.e. `offer` > `offers`, to keep consistent namespace in `psql` I decided to stay with `accessorie`